UnityProject/Assets/Scripts/Managers/Supporters/FetchSupporterMethods/FetchSupportersFromJsonInStreamingAssets.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using Logs;
+using Newtonsoft.Json;
+using SecureStuff;
+
+namespace Managers.Supporters.FetchSupporterMethods
+{
+    public class FetchSupportersFromJsonInStreamingAssets : IFetchSupporters
+    {
+        private const string FileName = "supporters.json";
+
+        public List<Supporter> FetchSupporters()
+        {
+            try
+            {
+                if (AccessFile.Exists(FileName, true, FolderType.Data))
+                {
+                    string jsonContent = AccessFile.Load(FileName, FolderType.Data);
+                    return JsonConvert.DeserializeObject<List<Supporter>>(jsonContent) ?? new List<Supporter>();
+                }
+
+                // If the file does not exist, create a default one
+                var defaultSupporters = new List<Supporter>();
+                defaultSupporters.Add( new Supporter
+				{
+					Identifier = "12345678901234567",
+					Flare = "[S] - "
+				});
```

Comment:
Is the flare supposed to be a prefix to appear in OOC? Because if that's the case, you just need to create the supporter role in permissions.toml and that's it.